src/gitmoji.js

```diff
@@ -149,13 +149,17 @@ class GitmojiCli {
 
     if (config.getAutoAdd()) {
       execa.stdout('git', ['add', '.'])
-        .then((res) => console.log(chalk.blue(res)))
-        .catch((err) => this._errorMessage(err.stderr))
+        .then((res) => {
+          console.log(chalk.blue(res))
+          return execa.shell(commit)
```

Comment:
You can return this directly from the arrow function

Reply:
Thanks a lot! I've changed it.